public/.htaccess

```diff
@@ -0,0 +1,12 @@
+<IfModule mod_rewrite.c>
```

Comment:
Ce fichier est-il vraiment nécessaire sur cette petite appli ?

Reply:
Pour mon petit hébergement pas cher, c’est necessaire pour activer le rewriting d’apache :)
Je peux l’ignorer de mon côté si tu veux. Mais il me semble que ça rend l’appli plus facile a installer dans ce cas, sans handicaper d’autres usages.

Reply:
Why not, pour les utilisateurs apache.